File: web/backend/src/models/dish.js
module.exports = (sequelize, DataTypes) => {
  const Dish = sequelize.define('Dish', {
    id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true,
    },
    name: {
      type: DataTypes.STRING,
      allowNull: false,
    },
    image: DataTypes.TEXT,
    portion_size: DataTypes.TEXT,
    technology: DataTypes.TEXT,
    preparation_time: DataTypes.INTEGER,
    dish_type_id: DataTypes.INTEGER,
    created_at: {
      type: DataTypes.DATE,
      defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
    },
  }, {
    tableName: 'dishes',
    timestamps: false,
  });
  return Dish;
};

File: web/backend/with_cash/app/models/dish.py
from sqlalchemy import Column, Integer, String, Text, ForeignKey, DateTime
from sqlalchemy.sql import func
from app.core.database import Base

class Dish(Base):
    __tablename__ = "dishes"
    id = Column(Integer, primary_key=True, index=True)
    name = Column(String, nullable=False, index=True)
    image = Column(Text)
    portion_size = Column(Text)
    technology = Column(Text)
    preparation_time = Column(Integer)
    dish_type_id = Column(Integer, ForeignKey("dish_types.id"))
    created_at = Column(DateTime, default=func.now())